.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: php
+
+dist: trusty
+
+env:
+  global:
+    - COMPOSER_ROOT_VERSION=2.x-dev
+
+matrix:
+  include:
+    - php: 5.6
+      env: PHPUNIT_TEST=core,cwpcore
+    - php: 7.0
+      env: PHPUNIT_TEST=admin,cwpcms,Default
+    - php: 7.1
+      env: PHPCS_TEST=1 PHPUNIT_COVERAGE_TEST=1
```

Comment:
We don't need to run coverage reports on recipes. PHPCS is good though!

Reply:
I left this because of the boilerplate style classes included with the recipe. But you're right, they have nothing to cover by these empty subclasses.